src/Sulu/Bundle/AdminBundle/Resources/js/components/Backdrop/README.md

```diff
@@ -7,7 +7,7 @@ intialState = {open: false};
 
 <div>
     <button onClick={() => setState({open: true})}>Open Backdrop</button>
-    <Backdrop isOpen={state.open} onClick={() => setState({open: false})} />
+    <Backdrop isOpen={!!state.open} onClick={() => setState({open: false})} />
```

Comment:
Because initially the `state.open` is `undefined` in styleguidist. As the default value for isOpen is `true` the Backdrop opens automatically. That's why the backdrops were open on page load all the time.